### PR TITLE
chore: Add a script for creating labels

### PR DESCRIPTION
**Pull request by sfc-gh-jcieslak**
The follow-up after new gh issue templates:
- A script for creating labels (if they don't exist already).
- A script for adding labels to issues (according to the prepared .csv - GitHub issues buckets).

### CODE REVIEW

**Review by sfc-gh-asawicki — APPROVED** [review]

File: pkg/scripts/issues/README.md [hunk 1]
```diff
@@ -26,4 +26,22 @@
...
+```
+2. Then, we have to get data about the existing issues with:
+```shell
+  cd gh && SF_TF_SCRIPT_GH_ACCESS_TOKEN=<YOUR_PERSONAL_ACCESS_TOKEN> go run .
+```
+3. Afterward, we need to process `issues.json` with:
+```shell
+  cd file && go run .
+```
+4. Next you have to analyze generated CSV and assign categories in the `Category` column and resource / data source in the `Object` column (the `GitHub issues buckets` Excel should be used here named as `GitHubIssuesBucket.csv`). Then, you'll be able to use this csv (put it next to the `main.go`) to assign labels to the correct issues.
```
[the remark this answers is not in the document]
Comment (sfc-gh-jcieslak) [reply pkg/scripts/issues/README.md:44]: added a brief explaination

File: pkg/scripts/issues/assign-labels/main.go [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+	"strconv"
+	"strings"
+)
+
+var lookupTable = make(map[string]string)
+
+func init() {
+	for _, label := range issues.RepositoryLabels {
+		parts := strings.Split(label, ":")
+		labelType := parts[0]
```
[the remark this answers is not in the document]
Comment (sfc-gh-jcieslak) [reply pkg/scripts/issues/assign-labels/main.go:22]: added a custom panic if len(parts) != 2

File: pkg/scripts/issues/README.md [hunk 3]
```diff
@@ -41,7 +41,7 @@
...
   cd file && go run .
 ```
-4. Next you have to analyze generated CSV and assign categories in the `Category` column and resource / data source in the `Object` column (the `GitHub issues buckets` Excel should be used here named as `GitHubIssuesBucket.csv`). Then, you'll be able to use this csv (put it next to the `main.go`) to assign labels to the correct issues.
+4. Next you have to analyze generated CSV and assign categories in the `Category` column and resource / data source in the `Object` column (the `GitHub issues buckets` Excel should be used here named as `GitHubIssuesBucket.csv`). The csv document be of a certain format with the following columns (with headers): "A" column with issue ID (in the format of "#<issue_id>"), "P" column with the category that should be assigned to the issue (should be one of the supported categories: "OTHER", "RESOURCE", "DATA_SOURCE", "IMPORT", "SDK", "IDENTIFIERS", "PROVIDER_CONFIG", "GRANTS", and "DOCUMENTATION"), and the "Q" column with the object type (should be in the format of the terraform resource, e.g. "snowflake_database"). Then, you'll be able to use this csv (put it next to the `main.go`) to assign labels to the correct issues.
```
Comment (sfc-gh-asawicki) [inline pkg/scripts/issues/README.md:44]: let's add this file (it's easier to understand and adjust)

**Review by sfc-gh-asawicki — APPROVED** [review]

File: pkg/scripts/issues/README.md (hunk 3 above)
Comment (sfc-gh-asawicki) [reply pkg/scripts/issues/README.md]: Hmmm, I hope d you will make this CSV much easier and limit it to only needed columns. I don't see a reason to keep the file with all this columns for this script (it complicates it unnecessarily).

**Review by sfc-gh-asawicki — APPROVED** [review]

**Pull request merged**